apps/site/assets/ts/schedule/components/line-diagram/LiveCrowdingIcon.tsx

```diff
@@ -1,26 +1,45 @@
 import React from "react";
 import { TooltipWrapper, crowdingIcon } from "../../../helpers/icon";
-import { crowdingDescriptions } from "../../../models/vehicle";
+import {
+  crowdingDescriptions,
+  crCrowdingDescriptions
+} from "../../../models/vehicle";
 import { CrowdingType } from "../__schedule";
 
 interface LiveCrowdingIconProps {
   crowding: CrowdingType;
+  tooltipPlacement?: "top" | "bottom" | "left" | "right";
+  isCommuterRail?: boolean;
 }
 
-const LiveCrowdingIcon = ({ crowding }: LiveCrowdingIconProps): JSX.Element => (
+const LiveCrowdingIcon = ({
+  crowding,
+  tooltipPlacement,
+  isCommuterRail
+}: LiveCrowdingIconProps): JSX.Element => (
   <div className="m-schedule-diagram__prediction-crowding">
     {crowding ? (
       <TooltipWrapper
-        tooltipText={`Currently <strong>${crowdingDescriptions(
-          crowding
-        ).toLowerCase()}</strong>`}
+        tooltipText={
+          isCommuterRail
+            ? `This train typically has<br /><strong>${crCrowdingDescriptions(
+                crowding
+              )}</strong>`
+            : `Currently <strong>${crowdingDescriptions(
+                crowding
+              ).toLowerCase()}</strong>`
+        }
         tooltipOptions={{
-          placement: "left",
+          placement: tooltipPlacement || "left",
           animation: false,
           html: true
         }}
       >
-        {crowdingIcon(`c-icon__crowding--${crowding}`)}
+        {crowdingIcon(
+          `c-icon__crowding--${crowding} ${
+            isCommuterRail ? "commuter-rail" : ""
```

Comment:
We have a `.commuter-rail` class name used elsewhere, I'm just afraid of collision... can we name this something more unique?